src/Missings.jl

```diff
@@ -165,4 +165,24 @@ function levels(x)
     levs
 end
 
+struct PassMissing{F} <: Function end
+
+(::PassMissing{F})(xs...;kw...) where {F} =
+     any(ismissing, xs) || any(ismissing, values(values(kw))) ? missing : F(xs...; kw...)
+
+"""
+    passmissing(f)
+
+Return a function that returns `missing` if any of its positional or keyword arguments
+are `missing` and otherwise applies `f` to those arguments.
+
+# Examples
+```jldoctest
+julia> passmissing(sqrt).([missing, 4])
```

Comment:
It would be useful to start with a simpler example with a single scalar (first non-missing, and then missing).

Reply:
added